src/core/utils/urlutils.js

```diff
@@ -82,3 +84,66 @@ export function equivalentParams (params1, params2) {
   }
   return true;
 }
+
+/**
+ * Creates a new params object without params that begin with the given prefixes
```

Comment:
How about "Creates a copy of the provided {@link SearchParams}, with the specified attributes filtered out"? The phrasing "Creates a new params object without params..." is a bit confusing.